### DISCUSSION TITLE
Question about Hubs Cloud / Reticulum release cycle

### DISCUSSION THREAD

**Discussion by CodingBitsDev** [mozilla/hubs#4644] [Hubs Cloud Questions] [answered]
Hey hubs team. I have some questions about the hubs clould release cycle.

With my current research I figured out this:

1. The hubs client will be updated via the hubs_cloud branch in this git
2. When the newest features are wanted we of course need to update the local client ourselfs.
3. The aws service is mostly updated outomaticly (unless there are major infrastructure changes)

In this regard there are some questions:

1. Can I be certain (or mostly certain, given that there is no unintended bug) that the hubs_cloud client won't break after an automatic update of the aws service?
2. The hubs client has the hubs_cloud branch for updates regarding the hubs_cloud service. Does an equivalent exist in the reticulum project?
3. If not is the master branch of reticulum the same that is deployed to aws as well as the official hubs instance?

**Comment by pattersonbl2** [comment, marked as answer]
1. We won't be able to guarantee an update won't affect existing Hubs cloud clients, but new code is tested before deployment. 
2. Reticulum has automatic updates using chef habitat, but any code changes will be also updated in Github. 
3. They are updated a little differently but the base code should be the same.
